steward/src/config.rs

```diff
@@ -72,48 +71,51 @@ impl StewardConfig {
 impl Default for StewardConfig {
     fn default() -> Self {
         Self {
-            keystore: "/tmp/keystore".to_owned(),
-            server: ServerSection::default(),
-            gravity: GravitySection::default(),
-            ethereum: EthereumSection::default(),
-            cosmos: CosmosSection::default(),
-            metrics: MetricsSection::default(),
             cellars: vec![CellarConfig::default()],
+            cosmos: CosmosSection::default(),
+            ethereum: EthereumSection::default(),
+            gravity: GravitySection::default(),
             keys: KeysConfig::default(),
+            metrics: MetricsSection::default(),
+            server: ServerSection::default(),
         }
     }
 }
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(default)]
 pub struct ServerSection {
-    pub address: Option<String>,
+    pub address: String,
     pub client_ca_cert_path: Option<String>,
-    pub port: Option<u16>,
+    pub port: u16,
     pub server_cert_path: String,
     pub server_key_path: String,
 }
 
 impl Default for ServerSection {
     fn default() -> Self {
         Self {
-            address: Some("0.0.0.0".to_owned()),
+            address: "0.0.0.0".to_string(),
             client_ca_cert_path: Some("".to_owned()),
-            port: Some(9999),
+            port: 5734,
             server_cert_path: "".to_owned(),
             server_key_path: "".to_owned(),
         }
     }
 }
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(default)]
 pub struct KeysConfig {
-    pub rebalancer_key: String,
+    pub delegate_key: String,
+    pub keystore: String,
 }
 
 impl Default for KeysConfig {
     fn default() -> Self {
         Self {
-            rebalancer_key: "".to_owned(),
+            delegate_key: "".to_owned(),
+            keystore: "/tmp/keystore".to_owned(),
```

Comment:
I think guessing a default keystore directory (as has been in the code for some time) is probably unhelpful, and `/tmp` is quite a poor place to store keys as it is not required to persist on system reboots.

Reply:
Hah I actually made an issue that cited this behavior as surprising if you forget your config arg. I'll change it to an empty string and make sure it gives a sensible error if you try to use it.